src/mgmt/deploy-dashboard-vm.py

```diff
@@ -179,9 +179,15 @@ def create_kickstart(ks_filename, cfg_filename):
     export https_proxy=${HTTP_Proxy}
 
     }
-
-  subscription-manager register --username ${SMUser} --password ${SMPassword} \
-${ProxyInfo}
+  
+if [[ -z ${SA_ip} ]]
+  then
+      subscription-manager register --username ${SMUser} --password ${SMPassword} ${ProxyInfo}
+  else
+      echo "$SA_ip    $SA_host" >> /etc/hosts
+      rpm -Uvh http://$SA_host/pub/katello-ca-consumer-latest.noarch.rpm
```

Comment:
Could this hard-coded URL be subject to change? If so, you might consider putting it in the settings file.